ui/helpers/constants.js

```diff
@@ -89,6 +89,7 @@ export const thPlatformMap = {
   'android-4-0-armv7-api16-beta': 'Android 4.0 API16+ Beta',
   'android-4-0-armv7-api16-release': 'Android 4.0 API16+ Release',
   'android-4-0-armv7-api16-ccov': 'Android 4.0 API16+ CCov',
+  'android-4-0-geckoview-fat-aar': 'Android 4.0 GeckoView multi-arch fat AAR',
```

Comment:
Fine by me.  The 4.0 part doesn't make a lot of sense, but it's consistent with the others.  Maybe `Android 4.0 API16+ GeckoView multi-arch fat AAR`, for even more consistency?  Anyway, it's better than nothing either way -- thank you!